pkg/services/authz/zanzana/schema/schema_folder.fga

```diff
@@ -4,15 +4,21 @@ type folder
   relations
     define parent: [folder]
 
-    # Action sets
-    define view: [user, service-account, team#member, role#assignee] or edit or view from parent
-    define edit: [user, service-account, team#member, role#assignee] or admin or edit from parent
+    # Permission levels
     define admin: [user, service-account, team#member, role#assignee] or admin from parent
+    define edit: [user, service-account, team#member, role#assignee] or edit from parent
+    define view: [user, service-account, team#member, role#assignee] or view from parent
+    define get: [user, service-account, team#member, role#assignee] or get from parent
+    define create: [user, service-account, team#member, role#assignee] or create from parent
+    define update: [user, service-account, team#member, role#assignee] or update from parent
+    define delete: [user, service-account, team#member, role#assignee] or delete from parent
+    define get_permissions: [user, service-account, team#member, role#assignee] or get_permissions from parent
+    define set_permissions: [user, service-account, team#member, role#assignee] or set_permissions from parent
 
-    define get: [user, service-account, team#member, role#assignee] or view or get from parent
-    define create: [user, service-account, team#member, role#assignee] or edit or create from parent
-    define update: [user, service-account, team#member, role#assignee] or edit or update from parent
-    define delete: [user, service-account, team#member, role#assignee] or edit or delete from parent
-
-    define get_permissions: [user, service-account, team#member, role#assignee] or admin or get_permissions from parent
-    define set_permissions: [user, service-account, team#member, role#assignee] or admin or set_permissions from parent
+    # Computed actions
+    define can_get: admin or edit or view or get
```

Comment:
Nice, I just wanted to comment that we still need some permissions like `create` to be assignable to users since we have then in roles:

```
| _user                      | relation        | object_type | object_id | condition_name     |
+----------------------------+-----------------+-------------+-----------+--------------------+
| role:basic_editor#assignee | create          | folder      | general   |                    |
```

Reply:
It was originally different